From 8152661f7dd127becc8f2c0b072be4c9ef69fc6a Mon Sep 17 00:00:00 2001
From: Yusuke Goda <yusuke.goda.sx@renesas.com>
Date: Thu, 21 Dec 2017 13:20:37 +0900
Subject: [PATCH 669/909] arm64: dts: r8a7795-h3ulcb: Add MMNGR driver node

commit d2af13285473a42214470294578babc0c4f310bb from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

This patch adds MMNGR driver for H3ULCB board on R8A7795 SoC.

Signed-off-by: Yusuke Goda <yusuke.goda.sx@renesas.com>
Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a7795-h3ulcb.dts | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a7795-h3ulcb.dts b/arch/arm64/boot/dts/renesas/r8a7795-h3ulcb.dts
index b9088a4e0a4a..47cadfd1fb9b 100644
--- a/arch/arm64/boot/dts/renesas/r8a7795-h3ulcb.dts
+++ b/arch/arm64/boot/dts/renesas/r8a7795-h3ulcb.dts
@@ -61,6 +61,11 @@
 			reg = <0x00000000 0x70000000 0x0 0x10000000>;
 		};
 	};
+
+	mmngr {
+		compatible = "renesas,mmngr";
+		memory-region = <&mmp_reserved>, <&lossy_decompress>;
+	};
 };
 
 &du {
-- 
2.17.1

